core/controller/.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!init.sh
+!build/distributions
```

Comment:
controller and invoker will need an exception for external libs and config, no? (per #3059)

Reply:
The external libs and config should be added by another Docker images which extends openwhisk images. So this dockerignore would not impact them.

As for default libs and config they would already be part of build/distributions/controller.tar. The Docker images in openwhisk only pull in files under build/distributions